src/scripts/main.js

```diff
@@ -18,10 +18,29 @@ const food = {
   },
 };
 
+const newTree = document.createElement('div');
+
+newTree.setAttribute('id', 'tree');
+
+document.body.prepend(newTree);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
```

Comment:
Creating the `ul` element outside of the loop is efficient. Ensure that any styles or attributes needed for the list are applied here.